docs/source/install.rst

```diff
@@ -12,15 +12,19 @@ Conda
 
 Dask is installed by default in `Anaconda <https://www.anaconda.com/download/>`_::
 
-You can update Dask using `conda <https://www.anaconda.com/download/>`_::
+You can update Dask using the `conda <https://www.anaconda.com/download/>`_: command:
 
-    conda install dask
+*   ``conda install dask``               : Installs dask from the default channel (recommended)
+*   ``conda install -c conda-forge dask``: Installs dask from `conda-forge <https://conda-forge.github.io/>`_.
 
-This installs Dask and all common dependencies, including Pandas and NumPy.
+This installs Dask and **all** common dependencies, including Pandas and NumPy.
```

Comment:
I like the emphasis here, especially given your experience with the bloated install.

Reply:
Cool, keeping it then :smile: